values.yaml

```diff
@@ -201,6 +201,27 @@ postgres:
   # the PostgreSQL connection. For acceptable values, see:
   #  https://www.postgresql.org/docs/9.1/libpq-ssl.html
   sslMode: "require"
+  # postgres.ssl -- Options for configuring the SSL cert, key, and root cert
+  # when connecting to Postgres.
+  ssl:
```

Comment:
what do you think about using `tls` here, makes things consistent with `ingress.tls` etc

Reply:
Postgres always refers to it as `SSL` so I was staying consistent with their documentation and naming.

Reply:
hmm. alright I guess. I get why they need to call it SSL (backward compatibility) but since this is a new field for us, I prefer TLS. not a big deal to me though.